src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -713,6 +721,27 @@ public boolean isAllFeedbackPathOptionsEnabled() {
         return true;
     }
     
+    public boolean isAllRecipientOptionsDisplayed(int questionNumber) {
+        List<WebElement> recipientOptions =
+                browser.driver.findElements(By.cssSelector("#recipienttype-" + questionNumber + " option"));
+        for (WebElement recipientOption : recipientOptions) {
+            String recipientOptionClass = recipientOption.getAttribute("class");
+            if ("hidden".equals(recipientOptionClass)) {
```

Comment:
HTML verification is an overkill. In that case, revert to the previous version (using `.hide()` and `.show()`) but use `element.getCssValue("display")` instead. Can't say that I like it, but it's the least fragile of all options.